Tests/PubNubContractTest/Steps/Files/PubNubFilesContractTestSteps.swift

```diff
@@ -156,5 +156,75 @@ public class PubNubFilesContractTestSteps: PubNubContractTestCase {
 
       self.wait(for: [sendFileExpect], timeout: 60.0)
     }
+    
+    When("^I send a file with '(.+)' space id and 'this-is-really-long-message-type-to-be-used-with-publish' message type$") { args, _ in
```

Comment:
@jguz-pubnub you can have multiple matches in step definition. I would suggest this:
```suggestion
    When("^I send a file with '(.+)' space id and '(.+)' message type$") { args, _ in
```

With code above, you get get last element from `args` and use it for `messageType`.

Reply:
Indeed, I checked it again and it's possible to have one template for 3 contracts and there's no need to repeat it:

```
@contract=sendFileWithSpaceIdAndMessageType
@contract=sendFileWithTooShortMessageType
@contract=sendFileWithTooLongMessageType
```